monitor/test/test_websock_server.py

```diff
@@ -33,8 +33,10 @@ def test_connection_handler(self):
 
         received_id = asyncio.Future()
         received_id.set_result("1234")
+        response = asyncio.Future()
+        response.set_result(None)
         websocket_mock.recv.return_value = received_id
-        websocket_mock.send.return_value = [None]
+        websocket_mock.send.return_value = response
```

Comment:
have you considered this?

```suggestion
        websocket_mock.send.return_value = asyncio.Future()
        websocket_mock.send.return_value.set_result(None)
```

It may read a bit better.

Reply:
I have, a bit more verbose. And it is just a test...